api/GPUAdapterInfo.json

```diff
@@ -224,7 +224,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": "26"
```

Comment:
Added via https://github.com/WebKit/WebKit/commit/c7c76cfe9bf7e8405b2be988389b35348ea48145 in WebKit [622.1.19](https://github.com/WebKit/WebKit/blob/c7c76cfe9bf7e8405b2be988389b35348ea48145/Configurations/Version.xcconfig#L24-L26) before Safari [26](https://github.com/mdn/browser-compat-data/blob/85b8827c1a9e4c77ea477d5bf29f7ad397e71696/browsers/safari.json#L370-L376).